src/app/server/Server.cpp

```diff
@@ -254,45 +254,52 @@ CHIP_ERROR Server::Init(AppDelegate * delegate, uint16_t secureServicePort, uint
     //
     // This is disabled for thread device because the same code is already present for thread devices in
     // src/platform/OpenThread/GenericThreadStackManagerImpl_OpenThread_LwIP.cpp
-    // https://github.com/project-chip/connectedhomeip/issues/14254
 #if !CHIP_DEVICE_CONFIG_ENABLE_THREAD
+    RejoinExistingMulticastGroups();
+#endif // !CHIP_DEVICE_CONFIG_ENABLE_THREAD
+
+exit:
+    if (err != CHIP_NO_ERROR)
     {
-        ChipLogProgress(AppServer, "Adding Multicast groups");
-        ConstFabricIterator fabricIterator = mFabrics.cbegin();
-        while (!fabricIterator.IsAtEnd())
-        {
-            const FabricInfo & fabric = *fabricIterator;
-            Credentials::GroupDataProvider::GroupInfo groupInfo;
+        ChipLogError(AppServer, "ERROR setting up transport: %s", ErrorStr(err));
+    }
+    else
+    {
+        ChipLogProgress(AppServer, "Server Listening...");
```

Comment:
Suggest adding which port it is listening

Reply:
done

Reply:
If you change the log, it breaks a cirque test since it expects this exact string